src/executables/util_cs.c

```diff
@@ -3275,51 +3280,60 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	    {
 	      goto check_applied_info_end;
 	    }
-	  error = db_restart (arg->command_name, TRUE, local_database_name);
-	  if (error != NO_ERROR)
+	  if ((error = db_restart (arg->command_name, TRUE, local_database_name)) != NO_ERROR)
 	    {
 	      goto check_applied_info_end;
 	    }
 
 	  if (HA_DISABLED ())
 	    {
+	      check_applied_info = false;
+	      printf ("\nERROR : ");
```

Comment:
`check_database_name()` function makes every error message from applyinfo utility printing "ERROR :" prefix impossible,

Because `check_database_name()` prints inside of its own function.

I will roll-back all error message format.